examples/lifetime_prediction/tests.py

```diff
@@ -38,9 +38,9 @@ def test_generate_lifetime_features(self):
         with h5py.File(h5py_hdf_path, "r") as global_data_h5f, \
                 h5py.File(data_bundle_hdf_path, "r") as data_bundle_h5f:
             assert (set(global_data_h5f)
-                    == set(data_definition.json() for data_definition in data_definitions))
+                    > set(data_definition.json() for data_definition in data_definitions))
```

Comment:
set(global_data_h5f) will have two more file than data_definitions from:
```
    @require('is_in_test_set', 'is_in_test_set_1126', random_state=1126)
    @require('is_in_test_set', 'is_in_test_set_5566', random_state=5566)
```

Reply:
then you should add them